src/error/index.js

```diff
@@ -0,0 +1 @@
+export * from './error';
```

Comment:
you file is `Error.js`, but you point out an `error` - why the names are different? It is not working for me.